doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20.rst

```diff
@@ -0,0 +1,445 @@
+
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
+
+* :ref:`fast_pair_locator_tag` sample:
+
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
+
+Edge Impulse samples
+--------------------
+
+* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target in the following samples:
+
+  * :ref:`ei_data_forwarder_sample`
+  * :ref:`ei_wrapper_sample`
+
+Matter samples
+--------------
+
+* Updated:
+
+  * All Matter samples that support low-power mode to enable the :ref:`lib_ram_pwrdn` feature.
+    It is enabled by default for the release configuration of the following samples:
+
+    * :ref:`matter_lock_sample`
+    * :ref:`matter_light_switch_sample`
+    * :ref:`matter_smoke_co_alarm_sample`
+    * :ref:`matter_window_covering_sample`
+
+  * All Matter samples to enable the ZMS file subsystem in all devices that contain MRAM, such as the nRF54H Series devices.
+
+* Disabled pausing Matter watchdog while CPU is in idle state in all Matter samples.
+  To enable it, set the :ref:`CONFIG_NCS_SAMPLE_MATTER_WATCHDOG_PAUSE_IN_SLEEP<CONFIG_NCS_SAMPLE_MATTER_WATCHDOG_PAUSE_IN_SLEEP>` Kconfig option to ``y``.
+
+* :ref:`matter_smoke_co_alarm_sample` sample:
+
+  * Added support for ICD dynamic SIT LIT switching (DSLS).
+
+SUIT samples
+------------
+
+* Updated the :ref:`suit_recovery` by adding support for triggering firmware recovery by pressing a hardware button or calling a dedicated API.
+  For more information, see :ref:`ug_nrf54h20_suit_recovery`.
+
+Other samples
+-------------
+
+* :ref:`coremark_sample` sample:
+
+  * Updated:
+
+    * Configuration for the :ref:`zephyr:nrf54h20dk_nrf54h20` board to support multi-domain logging using the ARM Coresight STM.
+    * The logging format in the standard logging mode to align it with the format used in the multi-domain logging mode.
+    * Support for alternative configurations to use the :ref:`file suffix feature from Zephyr <app_build_file_suffixes>`.
+      The following file suffixes are supported as alternative configurations:
+
+      * ``flash_and_run``
+      * ``heap_memory``
+      * ``static_memory``
+      * ``multiple_threads``
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Bluetooth libraries and services
+--------------------------------
+
+* Added the :ref:`rreq_readme` and :ref:`rrsp_readme` libraries.
+
+* :ref:`hogp_readme` library:
+
+  * Updated the :c:func:`bt_hogp_rep_read` function to forward the GATT read error code through the registered user callback.
+    This ensures that API user is aware of the error.
+
+* :ref:`bt_fast_pair_readme` library:
+
+  * Added support in build system for devices that do not support the :ref:`partition_manager`.
+    The :ref:`zephyr:nrf54h20dk_nrf54h20` board target is the only example of such a device.
+  * Updated the :c:func:`bt_fast_pair_info_cb_register` API to allow registration of multiple callbacks.
```

Comment:
This entry is general information and is not nRF54H20-specific. I think it should be kept in the latest changelog so that it appears in the next NCS (standard) release:

https://github.com/nrfconnect/sdk-nrf/pull/19597#discussion_r1893950771

Reply:
Removing info about bt_fast_pair_readme lib changes